netprofile_tickets/netprofile_tickets/views.py

```diff
@@ -803,7 +806,8 @@ def _send_ticket_mail(req, ticket=None, change=None):
 
     for sub in chain(state.subscriptions,
                      ticket.subscriptions,
-                     _default_subscriptions(ticket, change)):
+                     _default_subscriptions(ticket, change),
+                     old_state.subscriptions if old_state else []):
```

Comment:
_default_subscriptions() needs to come last, to prioritize more specific subscriptions/templates.